src/hotspot/share/compiler/compileBroker.hpp

```diff
@@ -301,7 +301,7 @@ class CompileBroker: AllStatic {
                                  int hot_count,
                                  CompileTask::CompileReason compile_reason,
                                  TRAPS);
-
+private:
```

Comment:
Making this helper method avoids anyone outside `CompileBroker` accidentally calling it.